### DISCUSSION TITLE
Tried to change email - locked out of only user account

### DISCUSSION THREAD

**Discussion by tunloop** [firefly-iii/firefly-iii#5879] [Firefly III] [answered]
So I tried to change the email, but forgot that my server does not have internet access nor a configured mail sending agent. 

Now I cant log into the account with the old or new email. I have root access and I see the user entry in mariadb, how can I go about unblocking the account so I can sign in with the new email?

Or better yet, can I just get rid of the email and use a normal username?

I am using mariadb 10.5 on debian 11.

**Comment by JC5** [comment]
There’s some hints how to fix this in previous discussions

**Reply by tunloop** [reply]
> There’s some hints how to fix this in previous discussions

Found one saying what to do, but I also have no clue how to change mariadb database entries. 

Quick googling, I've tried this command for the blocked_code:
ALTER TABLE users MODIFY blocked_code NULL;
which doesnt work.

but have no idea how to put numbers in there. 

Before I go and learn how to use mariadb, can someone just tell me what commands I need to run?

**Reply by JC5** [reply]
I'm sorry, there's not an easy escape hatch here.

**Reply by tunloop** [reply]
I've tried the following command:
ALTER TABLE users MODIFY blocked int DEFAULT 0;

Which returns query ok, but affects 0 rows. The entry for blocked doesn't change.

I am very lost here, all the results on google are highly advanced and only serve to confuse me more.

**Comment by tunloop** [comment, marked as answer]
Finally found the command, I have no idea why it was so hard to find.

UPDATE users SET blocked = "0" WHERE id = 1;
UPDATE users SET blocked_code = "NULL" WHERE id = 1;

Leaving that there in case anyone comes across this.
